src/main/java/software/amazon/awssdk/eventstreamrpc/ServiceOperationMappingContinuationHandler.java

```diff
@@ -13,10 +24,6 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import software.amazon.awssdk.crt.eventstream.*;
-
 public class ServiceOperationMappingContinuationHandler extends ServerConnectionHandler {
```

Comment:
This should be fixed in the CrtSmithyJavaCodegen package as well.

Reply:
yeah